content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -14,62 +14,68 @@ weight: 270
+you need to understand the typical expectations for operating an etcd cluster.
+Refer to the [etcd documentation](https://etcd.io/docs/) for more context.
 
-### Understanding etcdctl and etcdutl
+Key details include:
 
-`etcdctl` and `etcdutl` are command-line tools used to interact with etcd clusters, but they serve different purposes:
-
-- `etcdctl`: This is the primary command-line client for interacting with etcd over a
-network. It is used for day-to-day operations such as managing keys and values,
-administering the cluster, checking health, and more.
-
-- `etcdutl`: This is an administration utility designed to operate directly on etcd data
-files, including migrating data between etcd versions, defragmenting the database,
-restoring snapshots, and validating data consistency. For network operations, `etcdctl`
-should be used.
-
-For more information on `etcdutl`, you can refer to the [etcd recovery documentation](https://etcd.io/docs/v3.5/op-guide/recovery/).
-
-<!-- steps -->
-
-## Prerequisites
-
-* Run etcd as a cluster of odd members.
+* The minimum recommended etcd versions to run in production are `3.4.22+` and `3.5.6+`.
 
 * etcd is a leader-based distributed system. Ensure that the leader
   periodically send heartbeats on time to all followers to keep the cluster
   stable.
 
-* Ensure that no resource starvation occurs.
+* You should run etcd as a cluster with an odd number of members.
+
+* Aim to ensure that no resource starvation occurs.
 
   Performance and stability of the cluster is sensitive to network and disk
   I/O. Any resource starvation can lead to heartbeat timeout, causing instability
   of the cluster. An unstable etcd indicates that no leader is elected. Under
   such circumstances, a cluster cannot make any changes to its current state,
   which implies no new pods can be scheduled.
 
-* Keeping etcd clusters stable is critical to the stability of Kubernetes
-  clusters. Therefore, run etcd clusters on dedicated machines or isolated
-  environments for [guaranteed resource requirements](https://etcd.io/docs/current/op-guide/hardware/).
-
-* The minimum recommended etcd versions to run in production are `3.4.22+` and `3.5.6+`.
-
-## Resource requirements
+### Resource requirements for etcd
 
 Operating etcd with limited resources is suitable only for testing purposes.
 For deploying in production, advanced hardware configuration is required.
 Before deploying etcd in production, see
 [resource requirement reference](https://etcd.io/docs/current/op-guide/hardware/#example-hardware-configurations).
 
+Keeping etcd clusters stable is critical to the stability of Kubernetes
+clusters. Therefore, run etcd clusters on dedicated machines or isolated
+environments for [guaranteed resource requirements](https://etcd.io/docs/current/op-guide/hardware/).
+
+### Tools
+
+Depending on which specific outcome you're working on, you will need the `etcdctl` tool or the
```

Comment:
What does "outcome" here mean? Never saw this term anywhere else.

Reply:
I'm using it in a typical British English sense; see https://en.wikipedia.org/wiki/Outcomes_theory

I don't know 100% that this sense of the word is used in US English, but I think it's quite common.

Reply:
This does work in US English.